test/built-ins/String/prototype/codePointAt/position-equal-or-more-than-array-size-returns-undefined.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.3
+description: >
+  If pos >= size, return undefined
```

Comment:
This test seems redundant given `position-equal-or-more-than-size-returns-undefined.js`. I appreciate your interest in covering the Array case, but since there is no explicit "array branch" concerning the logic under test, it's probably better to leave this one out. I suppose we could keep this and instead duplicate all the other tests for the array case, but that seems excessive.